x-pack/test/functional/apps/rollup_job/hybrid_index_pattern.js

```diff
@@ -81,10 +81,11 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.settings.createIndexPattern(rollupIndexPatternName, '@timestamp', false);
 
       await PageObjects.settings.clickKibanaIndexPatterns();
-      const indexPattern = (await PageObjects.settings.getIndexPatternList()).pop();
-      const indexPatternText = await indexPattern.getVisibleText();
-      expect(indexPatternText).to.contain(rollupIndexPatternName);
-      expect(indexPatternText).to.contain('Rollup');
+      const indexPatternNames = await PageObjects.settings.getAllIndexPatternNames();
+      const filteredIndex = indexPatternNames.filter(
+        (i) => i.includes(rollupIndexPatternName) && i.includes('Rollup')
```

Comment:
I see this was pre-existing, but what is the reasoning for checking that the string also includes `Rollup`? Might be worth adding a comment in the code explaining it.